### ISSUE TITLE
support for influxdb auth

### ISSUE THREAD

**Issue by ZyanKLee** [boyney123/garie#19]
- `garie` version: 2.0-master

Relevant sections of code seem to be:

* https://github.com/boyney123/garie-lighthouse/blob/master/src/influx/influx.js
* https://github.com/boyney123/garie-pagespeed-insights/blob/master/src/influx/influx.js
* https://github.com/boyney123/garie-browsertime/blob/master/src/influx/influx.js

Problem description:

* garie plugins do not provide an option to set influxdb authentication data
* reusing an existing influxdb with http authentication enabled fails due to missing authentication data

Suggested solution:

* please add support for settings:

   * INFLUXDB_USER
   * INFLUXDB_USER_PASSWORD

   to be used while sending data to influxdb

**Comment by kafkiano** [comment]
I can only agree with you. I have already tried a workaround as well but right now I've just limited connections to localhost but database auth would be nice.

I also did not understand why theese variables are set in docker-compose.yml:

```
- INFLUXDB_ADMIN_USER=admin
- INFLUXDB_ADMIN_PASSWORD=admin
```

Because influxdb.conf states that admin auth is disabled:

```
[admin]
  enabled = true
  bind-address = ":8083"
  https-enabled = false
```
And i wonder if i need to limit bind-address to 127.0.0.1 here as well usually the ports setting in docker-compose should be enough?
